src/App.css

```diff
@@ -102,9 +92,16 @@ main > h1 + * {
   margin-top: 1rem;
 }
 
-.map__inner {
-  width: 100%;
+.view__content {
+  overflow-y: auto;
+}
+
+.map {
+  display: flex;
```

Comment:
do we need to `flex` here? what did you encounter that led you to flex the map?

Reply:
We don't, you're right, it's fixed now!😅